core/2d/CCParticleSystem.cpp

```diff
@@ -1,3 +1,7 @@
+#include "CCParticleSystem.h"
+#include "CCParticleSystem.h"
+#include "CCParticleSystem.h"
```

Comment:
duplicated includes?

Reply:
They're all not intended, imma remove them, everytime i create a function definition in visual studio it adds an include.. which kinda sucks 😕